straxen/rundb.py

```diff
@@ -12,7 +12,7 @@
 export, __all__ = strax.exporter()
 
 
-default_mongo_url = 'fried.rice.edu:27017/xenonnt'
+default_mongo_url = 'xenon-rundb.grid.uchicago.edu:27017,fried.rice.edu:27017,xenon1t-daq.lngs.infn.it:27017/xenonnt'
```

Comment:
Would it make sense to put fried.rice first? I'm not an expert but I can imagine that it tries connecting sequentially?

Reply:
If it goes sequentially I think it makes the most sense to have the Chicago one first since that is the closest to midway and on same uchicago network.